src/index-metadbs/mpds/v1/links.json

```diff
@@ -0,0 +1,34 @@
+{
+   "meta" : {
+       "api_version" : "1.0.0",
+       "query" : {
+           "representation" : "/links"
+       },
+       "more_data_available" : false,
+       "schema" : "https://schemas.optimade.org/openapi/v1.0/optimade_index.json"
+   },
+   "data" : [
+      {
+         "id" : "mpds",
+         "type" : "links",
+         "attributes" : {
+            "name": "Materials Platform for Data Science",
+            "description": "A highly curated Pauling File dataset based on ~0.5M publications and backing up Springer Materials, ICDD PDF, ASM APD, MedeA, Pearson Crystal Data, AtomWork Advanced, etc.",
+            "base_url": "https://api.mpds.io",
+            "homepage": "https://mpds.io",
+            "link_type": "child"
+         }
```

Comment:
As this API is private, it would be helpful to [disable aggregation by clients](https://github.com/Materials-Consortia/OPTiMaDe/blob/develop/optimade.rst#link-aggregate-options) by adding `"aggregate": "no"`.

This will also stop the automatic validator from running daily for MPDS. 

You could then potentially add `"no-aggregate-reason"` with instructions for how to get an API key?

Reply:
Note: the comment on aggregation may still apply as the commercial nature of some required fields (e.g. `elements`) may break clients